keras_nlp/samplers/top_k_sampler.py

```diff
@@ -63,10 +63,12 @@ def __init__(
         self,
         k=5,
         seed=None,
+        temperature=1.0,
```

Comment:
if you do the changes suggested above, you won't have to do anything beyond passing a `**kwargs` through.